akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/crdt/CounterSpec.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2017-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.crdt
+
+import akka.actor.typed.ActorRef
+import akka.actor.typed.scaladsl.Behaviors
+import akka.persistence.testkit.query.scaladsl.PersistenceTestKitReadJournal
+import akka.persistence.typed.crdt.CounterSpec.PlainCounter.{ Decrement, Get, Increment }
+import akka.persistence.typed.scaladsl.{ ActiveActiveEventSourcing, Effect, EventSourcedBehavior }
+import akka.persistence.typed.{ ActiveActiveBaseSpec, ReplicaId }
+
+object CounterSpec {
+
+  object PlainCounter {
+    sealed trait Command
+    case class Get(reply: ActorRef[Long]) extends Command
+    case object Increment extends Command
+    case object Decrement extends Command
+  }
+
+  import ActiveActiveBaseSpec._
+
+  def behavior(
```

Comment:
Not really important but styleguide-nitpick, should be `PlainCounter.apply(...)` rather than `behavior(...)`